components/Speaker.vue

```diff
@@ -0,0 +1,37 @@
+<template lang="pug">
+    .speaker(@click="modalActivate")
+        .speaker-img-container
+            img.speaker-img(:src="require(`~/assets/${speaker.speakerImage}`)")
```

Comment:
`test-speaker-in-cms.json` breaks build because of image url: `/images/uploads/Profile_Pic_BW.jpg`
Path needs to not have `/` prefix for file path resolution.